tests/integration/conftest.py

```diff
@@ -467,11 +491,17 @@ def configure_environment(get_configuration, request):
                 delete_registry(registry_parser[match.group(1)], match.group(2), KEY_WOW64_32KEY)
                 delete_registry(registry_parser[match.group(1)], match.group(2), KEY_WOW64_64KEY)
 
+    error_in_configuration = False
     if sys.platform == 'win32':
-        control_service('start')
-
+        try:
+            control_service('start')
+        except ValueError:
+            error_in_configuration = True
+            pass
```

Comment:
Same here. I prefer to raise an exception if the daemon can't start and stop the test.